changelog/20250925_fix_use_cluster_domain_environment_variable_instead_of.md

```diff
@@ -0,0 +1,14 @@
+---
+kind: feature
```

Comment:
while I agree this is a feature, the changelog filename should be renamed to:
`changelog/20250925_feature_use_cluster_domain_environment_variable.md`

Also the PR description is not up to date